components/edit_channel_header_modal/index.js

```diff
@@ -9,10 +9,14 @@ import {Preferences} from 'mattermost-redux/constants';
 
 import {closeModal} from 'actions/views/modals';
 
+import {setShowPreviewOnEditChannelHeaderModal} from '../../actions/views/rhs';
+import {showPreviewOnEditChannelHeaderModal} from '../../selectors/rhs';
```

Comment:
Can you change these imports to be absolute and not relative? We only usually use relative imports for files in the same folder.